Python/optimizer_bytecodes.c

```diff
@@ -1088,6 +1088,10 @@ dummy_func(void) {
         res = sym_new_type(ctx, &PyLong_Type);
     }
 
+    op(_GET_LEN, (-- len)) {
+        len = sym_new_type(ctx, &PyLong_Type);
```

Comment:
If you want to improve this in a follow-up PR, you could narrow this to a constant for constant strings and constant/abstract tuples.